bin/mail/2016-05-25-grassroots-congress/send-real-emails.txt

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+#20854
+../send-to-mailing-list --limit 21000 2016-5-24-WA-teacher-Senator ./wa-teachers-deduped.csv
```

Comment:
Yes. I think they set that one up yesterday, and the others today. Anyway that is the correct template name.